graphql-dgs-spring-boot-oss-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/autoconfig/DgsAutoConfiguration.kt

```diff
@@ -166,20 +168,42 @@ open class DgsAutoConfiguration(
         return Executors.newSingleThreadScheduledExecutor()
     }
 
+    @Bean
+    @ConditionalOnProperty(
+        prefix = "$AUTO_CONF_PREFIX.convertAllDataLoadersToWithContext",
+        name = ["enabled"],
+        havingValue = "true",
+        matchIfMissing = true
+    )
+    @Order(0)
```

Comment:
Consider using something like ` @Order(PriorityOrdered.HIGHEST_PRECEDENCE)`

Reply:
Related to what @srinivasankavitha mentions, 0 is neither highest or lowest, because priority is INT.max/INT.min.

Reply:
see: https://github.com/Netflix/dgs-framework/pull/1818#discussion_r1499974931